cli/cmd/proxy/loglevel/command.go

```diff
@@ -222,7 +227,7 @@ func (l *LogLevelCommand) fetchAdminPorts() (map[string]int, error) {
 	return adminPorts, nil
 }
 
-func (l *LogLevelCommand) fetchLogLevels(adminPorts map[string]int) (map[string]LoggerConfig, error) {
+func (l *LogLevelCommand) fetchOrSetLogLevels(adminPorts map[string]int) error {
```

Comment:
so I also consolidated the calls to envoy into this function, because they were both the exact same code, I'd prefer to keep them separate so it reads better, but having the envoy API is kinda forcing my hand to keep this code from being completely duplicated